builtin/game/init.lua

```diff
@@ -34,3 +34,7 @@ dofile(gamepath.."forceloading.lua")
 dofile(gamepath.."statbars.lua")
 
 profiler = nil
+
+core.register_on_mods_loaded(function()
+	core.log("Server mods loaded.")
```

Comment:
Seems superfluous. The mod loading procedure is already logged as verbose (or info?).

Reply:
i can remove it if wanted, at least it permits to show it works as intended :)
Do i need to keep it for CSM mods ?